docs/api/groups.rst

```diff
@@ -3,185 +3,237 @@
 Groups
 ######
 
+A group associates a name to a list of principals. It is useful in order to
+handle permissions. Groups are handled in buckets.
+
 A group is a mapping with the following attributes:
 
 * ``members``: a list of :term:`principals <principal>`
 * ``permissions``: (*optional*) the :term:`ACLs <ACL>` for the group object
+  (e.g who has the rights to read or update the group itself.)
+
+Creating a group
+================
 
+.. http:post:: /buckets/(bucket_id)/groups
 
-POST /buckets/<bucket_id>/groups
-================================
+    **Requires authentication**
 
-**Requires authentication**
+    Creates a new bucket's group with a generated id.
```

Comment:
bucket group (?)

AFAIK ``'s`` will express possession, not sure it works here